javascript/big-practice/src/scripts/utils/fetchAPI.js

```diff
@@ -1,6 +1,45 @@
 import { API_MESSAGE } from '../constants/message';
 
 const URL = 'http://localhost:3000/notes';
+const userURL = 'http://localhost:3000/users';
+
+/**
+ * @description function get list notes with username of user
+ *
+ * @param {String} username get from input login
+ *
+ * @returns {Object} user
+ */
+export const getUsersByUsername = async (username) => {
```

Comment:
You are getting a user by username, so please help check the naming convention here.